src/renderer/views/login/MiningView.tsx

```diff
@@ -15,15 +15,15 @@ const MiningView = observer(
     const classes = miningViewStyle();
     const setMining = (isMining: boolean) => {
       routerStore.push("/lobby/preload");
-      if (
-        !ipcRenderer.sendSync(
-          "standalone/set-private-key",
-          accountStore.privateKey
-        )
-      ) {
+      const setSucceed: boolean = ipcRenderer.sendSync(
+        "standAlone/set-private-key",
```

Comment:
It seems there is no changes about chanel renaming but it was changed into `standAlone`.

https://github.com/planetarium/9c-launcher/blob/ca76d73a0acaf55effad77cfed28704f7f53a939/src/main/standalone.ts#L39

Is it intended?